### DISCUSSION TITLE
LGSL v6.2.0

### DISCUSSION THREAD

**Discussion by tltneon** [tltneon/lgsl#117] [General]
[![GitHub followers](https://img.shields.io/github/followers/tltneon?style=for-the-badge)](https://github.com/tltneon?tab=followers)
[![GitHub stars](https://img.shields.io/github/stars/tltneon/lgsl?style=for-the-badge)](https://github.com/tltneon/lgsl/stargazers)
[![GitHub forks](https://img.shields.io/github/forks/tltneon/lgsl?style=for-the-badge)](https://github.com/tltneon/lgsl/fork)
[![Packagist](https://img.shields.io/packagist/l/tltneon/lgsl?style=for-the-badge)](https://github.com/tltneon/lgsl/blob/master/LICENSE)
- **Added Preloader feature**
	- *Loading animation by (https://codepen.io/Bilal1909)*
- **Added Factorio, Black Ops 3, S.T.A.L.K.E.R. COP, Mafia 2: MP, Just Cause 2: MP support**
- **Added Satisfactory, World of Warcraft, Farming Simulator support (only online-offline status)**
- **Fixed FiveM, Battlefield 3, TeamSpeak3/TeaSpeak, Minecraft protocols**
- **New game icons**
- **Userbar & Charts improvements**
- **Romanian language**
	- *Thanks to [@BaxAndrei](https://github.com/BaxAndrei)*
- **Chinese simplified language**
	- *Thanks to [@malimaliao](https://github.com/malimaliao)*
- **Minor fixes**
- * Tested on PHP 8.2.0-dev

<hr /><em>This discussion was created from the release <a href='https://github.com/tltneon/lgsl/releases/tag/v6.2.0'>LGSL v6.2.0</a>.</em>

**Comment by Webby0815** [comment]
Thanks for the new Version , [G-Tracker](http://g-tracker.de/) is now on 6.2.0 !!! Works Great
